src/lib/descriptor/mod.rs

```diff
@@ -638,5 +638,13 @@ pub fn load(
         }
     }
 
+    for (name, _) in config.tasks.iter() {
+        match name.as_str() {
+            "before_each" => config.config.before_each_task = Some(name.to_string()),
+            "after_each" => config.config.after_each_task = Some(name.to_string()),
+            _ => {}
+        }
```

Comment:
what does this loop actually do? what does it mutate?
meaning are you setting config.before_each_task="before_each"? if so why loop? why not just configure it? why the naming magic?

Reply:
I couldn't figure out how to else get the `before_each_task` and `after_each_task` to be populated. Tried a bunch of solutions, this is the only one that seemed to make it accessible to future locations (specifically `src/lib/execution_plan.rs`).